docs/source/api/config/config.rst

```diff
@@ -0,0 +1,22 @@
+`Config`
+============
+
+| Config determines how a session should run in tf-encrypted.
+| There are two primary ways in which config can be used:
+.. TODO: How do we link these to the real thing?
+.. py:class:: tensorflow_encrypted.config.LocalConfig
+and
+
+.. py:class:: tensorflow_encrypted.config.RemoteConfig
+As the name implies, `LocalConfig` is used to create a
+local session. This is useful for quick debugging and prototyping.
```

Comment:
Perhaps instead of local session we should define what that means: to run an entire session locally on your compute where as RemoteConfig is used to specify how to run a session in a distributed setting (e.g. using multiple machines).